packages/form-builder/src/components/FormField/types.d.ts

```diff
@@ -5,9 +5,16 @@ export interface PeriodValue {
 
 export type FieldItemValue = string | number | null | undefined;
 
+export type OtherValue = string | null;
+
 export type ChoiceValue = FieldItemValue | FieldItemValue[];
 
-export type FieldValue = string | number | null | undefined | PeriodValue | ChoiceValue;
+export interface IChoiceValue {
+	value: ChoiceValue;
+	other?: OtherValue;
+}
+
+export type FieldValue = string | number | null | undefined | PeriodValue | IChoiceValue;
```

Comment:
Comme maintenant le champ de type 'select' possède un retour contenant potentielement 2 valeurs avec le champ autre. Je ne sais pas trop comment faire la différence entre les valeurs de retour du champ complet avec la valeur autre et le champ de choix sans le autre.

J'hésite donc à renommer plusieurs types ici, qu'en pensez vous ?

```suggestion
export interface ChoiceValue {
	value: ChoiceFieldValue;
	other?: OtherFieldValue;
}

export type FieldValue = string | number | null | undefined | PeriodValue | ChoiceValue;
```

Reply:
le renommage me va !